src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -1,56 +1,52 @@
-import * as hooks from "actions/hooks";
-import React, {useMemo} from "react";
+import React from "react";
 import {Link} from "react-router-dom";
 import PropTypes from "prop-types";
-import {useSelector} from "react-redux";
 
 import ConfirmIcon from "@material-ui/icons/CheckCircle";
 import Grid from "@material-ui/core/Grid";
-import Loading from "components/OmouComponents/Loading";
 import Paper from "@material-ui/core/Paper";
 import Typography from "@material-ui/core/Typography";
 import UnconfirmIcon from "@material-ui/icons/Cancel";
-import LoadingError from "./LoadingCourseError";
 import Moment from "react-moment";
 
-import {courseDateFormat} from "utils";
+import gql from "graphql-tag";
+import { useQuery } from "@apollo/react-hooks";
 
-export const GET_INSTRUCTOR_ENROLLMENTS = `
-  query InstructorEnrollments($instructorId) { 
-    courses(filter: { instructor.user.id: $instructorId){
+export const GET_INSTRUCTOR_ENROLLMENTS = gql`
+  query InstructorEnrollments($instructorID: ID!) { 
+    courses(instructorId: $instructorID) {
+      id
+      dayOfWeek
+      description
+      startDate
+      startTime
+      endDate
+      endTime
+      isConfirmed
       title
     }
   }
 `;
 
 const InstructorCourses = ({ instructorID }) => {
-  const courses = useSelector(({ Course }) => Course.NewCourseList);
-  const courseStatus = hooks.useCourse();
 
-  const courseIDs = useMemo(
-    () =>
-      Object.keys(courses).filter(
-        (courseID) => instructorID === courses[courseID].instructor_id
-      ),
-    [courses, instructorID]
-  );
+  const { loading, error, data } = useQuery(GET_INSTRUCTOR_ENROLLMENTS, {
+		variables: {instructorID}
+	})
 
-  if (Object.keys(courses).length === 0) {
-    if (hooks.isLoading(courseStatus)) {
-      return <Loading />;
-    }
-    if (hooks.isFail(courseStatus)) {
-      return <LoadingError error="courses" />;
-    }
-  }
+	if (loading ) return null;
+
+  if (error ) return "Unable to load courses";
```

Comment:
There should be an <Error/> component as well, not sure, but double check

Reply:
Replaced the string with <Error/>